web-ng/src/app/components/layer-list-item/layer-list-item.component.ts

```diff
@@ -31,16 +35,19 @@ export class LayerListItemComponent implements OnInit, OnDestroy {
   @Input() layer?: Layer;
   @Input() actionsType: LayerListItemActionsType =
     LayerListItemActionsType.MENU;
+  projectId?: string | null;
```

Comment:
Do we still need `| null` with `?`?

Reply:
yes we do need | null with ? as in the router service we are projectId?: string | null because paramap returns an observable of string | null type.